features/booking/persist_data.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Extract booking data
```

Comment:
This doesn't describe persisting data. Again, why are we introducing new acceptance criteria?